pom.xml

```diff
@@ -32,6 +40,26 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
```

Comment:
Pytanie czy to jest w ogóle potrzebne... spring-boot-starter-test powinien to dostarczyć